doc/TUTORIAL.md

```diff
@@ -96,32 +93,35 @@ Each service has a directory for its files:
 All service badge classes inherit from [BaseService] or another class which extends it.
 Other classes implement useful behavior on top of [BaseService].
 
-* [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
+- [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
```

Comment:
Ah just realized these links are bad now, should be `https://github.com/badges/shields/blob/master/core/base-service/base-json.js`